src/main/resources/mapper/otherMappers/ClassesSubscriptions_mapper.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE mapper
+        PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN"
+        "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+<mapper namespace="ClassesSubscriptions_mapper">
```

Comment:
is that additional table for m:n? if so do not define mappers for such kind of tables